src/main/java/io/crate/client/jdbc/CrateConnection.java

```diff
@@ -38,11 +38,13 @@
     private CrateDatabaseMetaData metaData;
     private String databaseVersion;
     private boolean strict;
+    private Properties properties;
 
     public CrateConnection(ClientHandleRegistry.ClientHandle handle, Properties properties) {
         this.clientHandle = handle;
         this.readOnly = false;
         this.strict = Boolean.valueOf(properties.getProperty("strict", "false"));
```

Comment:
`strict` is also a property and should be handled the same way all others are handled